metametamodel/metametamodel.adoc

```diff
@@ -378,13 +378,74 @@ An EnumerationLiteral is similar to Ecore’s `EEnumLiteral` and to MPS’ `Enum
 .Characteristics
 An EnumerationLiteral has a <<INamed.name, `name`>> and an <<IKeyed.key, `key`>>.
 
-An Enumeration is a <<IKeyed>> (as it has a key) and indirectly an <<INamed>> (as it has a name).
+An EnumerationLiteral is a <<IKeyed>> (as it has a key) and indirectly an <<INamed>> (as it has a name).
 
 .Constraints
 Each EnumerationLiteral must have a unique `name` within the Enumeration.
 
 Each EnumerationLiteral belongs to one and only one Enumeration.
 
+[[StructuredDatatype]]
+==== StructuredDatatype
+Represents a collection of named instances of <<PrimitiveType, Primitive Types>>.
```

Comment:
I would add a comment to explain the difference from a Concept. For me the main difference is in the identify: instances of StructuredDatatype have none. They are just values, and if two StructuredDatatype have the same values for the same fields they are interchangeable, while the same is not true for nodes

Reply:
@dslmeinte to adjust

Reply:
See https://github.com/LionWeb-io/specification/pull/282 / https://github.com/LionWeb-io/specification/pull/282/commits/b527aa1158789b545abfd4069189cc67f933d4d5